modules/setting/mailer.go

```diff
@@ -151,6 +152,22 @@ func loadMailerFrom(rootCfg ConfigProvider) {
 		log.Fatal("Unable to map [mailer] section on to MailService. Error: %v", err)
 	}
 
+	overrideHeader := rootCfg.Section("mailer.override_header").Keys()
+	MailService.OverrideHeader = make(map[string][]string)
+	for _, key := range overrideHeader {
+		keyName := key.Name()
+		values := key.Strings(",")
```

Comment:
What is this `Strings` method? Never seen it and it's hard to google for.

Reply:
it returns `[]string{}` type cut by delimiter `,`

Reply:
Ah I see it's not a standard golang function, but something defined on the config keys.